server/models/Product.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+
+const productSchema = new mongoose.Schema({
+    name: {
```

Comment:
I don't know if this is required, but shouldn't there be a field for product image URL, since that's one of the pieces of info we scrap.

Reply:
i think it's good point. @sina-jamshidi ?

Reply:
i was actually thinking about this too while working on the scrapper

Reply:
I'll will add productImage field in the interface (client side)